packages/cli/src/open/open.js

```diff
@@ -39,7 +39,7 @@ async function runCommand(options) {
   }
 
   for (const lhr of representativeLhrs) {
-    if (options.url && lhr.finalUrl !== options.url) continue;
+    if (options.url && (lhr.finalUrl !== options.url && !options.url.some(item => lhr.finalUrl.includes(item)))) continue;
```

Comment:
how about a target URLs set? I think this will fail when url is a string, not an array

`const targetUrls = new Set([].concat(options.url || []));`

```suggestion
    if (targetUrls.size && !targetUrls.has(lhr.finalUrl)) continue;
```

Reply:
@patrickhulce Thanks for the swift feedback! I did actually test my solution with both string and array and it does work with both. However, I'd defer to you as creator/maintainer on which approach is best. I've quickly tested your solution locally and it also works as expected 👍 

I assume `targetUrls` is best set on line 40 after the `representativeLhrs` check?

Reply:
> it does work with both

Only when the `finalUrl` of all the reports matches `options.url` ;) `TypeError: options.url.some is not a function` if you have an LHR with a non-matching URL that should be skipped and `options.url` is a string

> I assume targetUrls is best set on line 40 after the representativeLhrs check?

Yep SGTM, thanks! 👍